src/components/Support/Support.jsx

```diff
@@ -32,7 +33,7 @@ const Support = () => {
           ____zł
         </button>
       </div>
-      <Button type="submit" content={t('support.button')} variant="primary" />
+      <Button type="submit" content="Wesprzyj" variant="primary" />
```

Comment:
tu trzeba przywrócić to co usunięte

Reply:
A przepraszam, bo podmieniłem sobie, aby sprawdzić działanie kody i potem źle przywróciłem ;g sorry. Poprawiam.